scripts/utils.js

```diff
@@ -117,3 +117,30 @@ export const getIconsData = async () => {
  */
 export const getDirnameFromImportMeta = (importMetaUrl) =>
   path.dirname(fileURLToPath(importMetaUrl));
+
+/**
+ * Get information about third party extensions.
+ */
+export const getThirdPartyExtensions = async () => {
+  const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
```suggestion
  const __dirname = getDirnameFromImportMeta(import.meta.url);
```
in the test file too